modules/terraform/cmd.go

```diff
@@ -16,6 +16,15 @@ func GetCommonOptions(options *Options, args ...string) (*Options, []string) {
 	if options.NoColor && !collections.ListContains(args, "-no-color") {
 		args = append(args, "-no-color")
 	}
+
+	if options.TerraformBinary == "" {
+		options.TerraformBinary = "terraform"
+	}
```

Comment:
Hmm, not sure if we want a side effect here. It would seem a bit unexpected for a `Get*` function to edit the object. OTOH, this is a benign change since we don't really expect this to be empty string.

Can you add a comment to the function docs stating that this will initialize `options.TerraformBinary` if it is not set?